max2.js

```diff
@@ -0,0 +1,42 @@
+function secondMax(arr){
```

Comment:
Even when corrected this solution is inefficient, we can suppose `Math.max` will traverse the entire array, in your current implementation this happens 3 times.

Can you come up with a solution that only traverses the array 1 time.